tests/integration_test.rs

```diff
@@ -0,0 +1,42 @@
+#![warn(clippy::pedantic)]
+
+use libcnb_test::{BuildpackReference, IntegrationTest};
+use std::io;
+
+#[test]
+#[ignore]
+fn test() {
+    IntegrationTest::new("heroku/buildpacks:20", "tests/fixtures/default_ruby")
+        .buildpacks(vec![BuildpackReference::Crate])
+        .run_test(|context| {
+            // On failure, print the stdout
+            println!("{}", context.pack_stdout);
+
+            assert!(context
+                .pack_stdout
+                .contains("---> Download and extracting Ruby"));
+            assert!(context.pack_stdout.contains("Running: bundle install"));
```

Comment:
`libcnb-test` now comes with `assert_contains!` which provides better UX for this case when the assertion fails (it prints the expected vs actual values rather than just an "expected true, got false").

See:
https://github.com/Malax/libcnb.rs/blob/424cdf94783cd2da10125c1edea540ddcfe6fdc0/examples/execd/tests/integration_test.rs#L21

(there are a few other usages here and below that could use it)